lib/pages/main_page.dart

```diff
@@ -180,45 +183,110 @@ class _MainPageState extends State<MainPage>
           if (snapshot.connectionState != ConnectionState.done) {
             return const SizedBox.shrink();
           }
-          return CustomScrollView(
-            controller: _scrollController,
-            scrollBehavior: const ScrollBehavior()..copyWith(overscroll: false),
-            slivers: [
-              SliverAppBar(
-                floating: true,
-                forceElevated: true,
-                // scrolledUnderElevation: v ? 2 : 1,
-                snap: true,
-                pinned: false,
-                actions: [
-                  IconButton(
-                    tooltip: "Next Para",
-                    icon: const Icon(Icons.arrow_back),
-                    onPressed: () {
+          return KeyboardListener(
```

Comment:
I think https://docs.flutter.dev/ui/interactivity/actions-and-shortcuts should be used

Reply:
@Waqar144 Migrated everything to `CallbackShortcuts`. Didn't feel like the use case calls for all the complexity that comes with Intent & Actions so just used CallbackActions for now.